draft-raza-ace-cbor-certificates.md

```diff
@@ -116,11 +116,11 @@ CBOR certificates are defined in terms of RFC 7925 profiled X.509 certificates:
 
 * version. The 'version' field is known (fixed to v3), and is omitted in the CBOR encoding.
 
-* serialNumber. The 'serialNumber' field is encoded as a CBOR byte string. 
+* serialNumber. The 'serialNumber' field is encoded as a CBOR byte string.
 
 * signature. The 'signature' field is always the same as the 'signatureAlgorithm' field and always omitted from the CBOR encoding.
 
-* issuer. In the general case, the Distinguished Name is encoded as CBOR map, but if only CN is present the value can be encoded as a single text value.
+* issuer. In the general case, the Distinguished Name is encoded as CBOR map. The value is represented as an array, including the original byte value and the value type used for canonical recomposition. If only CN is present the value can be encoded as a single byte or text value.
```

Comment:
The original DER SEQUENCE has an ordering.
While I accept that CBOR MAPs can be ordered, are we allowed to depend upon this?
A CBOR array would be less ambiguous.